go/pkg/client/cas.go

```diff
@@ -1232,34 +1232,35 @@ func (c *Client) DownloadDirectory(ctx context.Context, d digest.Digest, execRoo
 		return nil, stats, err
 	}
 
-	outStats, err := c.downloadOutputs(ctx, outputs, execRoot, cache)
+	outStats, err := c.downloadOutputs(ctx, outputs, root, cache)
 	stats.addFrom(outStats)
 	return outputs, stats, err
 }
 
 // DownloadActionOutputs downloads the output files and directories in the given action result. It returns the amount of downloaded bytes.
 // It returns the number of logical and real bytes downloaded, which may be different from sum
 // of sizes of the files due to dedupping and compression.
-func (c *Client) DownloadActionOutputs(ctx context.Context, resPb *repb.ActionResult, execRoot string, cache filemetadata.Cache) (*MovedBytesMetadata, error) {
+func (c *Client) DownloadActionOutputs(ctx context.Context, resPb *repb.ActionResult, execRoot, workingDir string, cache filemetadata.Cache) (*MovedBytesMetadata, error) {
```

Comment:
Same, `execRoot, workingDir` -> `outDir`.

This function doesn't need to worry about these RE API details.

Reply:
Done.